mlflow/metrics/genai/model_utils.py

```diff
@@ -9,9 +10,32 @@
 _logger = logging.getLogger(__name__)
 
 
+def get_endpoint_type(endpoint_uri: str) -> Optional[str]:
```

Comment:
note: just extracted from the original `score_model_on_payload` function to handle new requirement